src/modules/zerion-api/zerion-api.client.ts

```diff
@@ -1,9 +1,19 @@
+import { openTurnstileWidgetIfNeeded } from 'src/ui/features/turnstile/helpers';
 import { getAddressProviderHeader } from './requests/shared.client';
 import type { ZerionApiContext } from './zerion-api-bare';
 import { ZerionApiBare } from './zerion-api-bare';
 
 const context: ZerionApiContext = {
   getAddressProviderHeader,
+  getKyOptions: () => ({
+    hooks: {
+      afterResponse: [
+        (_, __, response) => {
```

Comment:
double underscore %)

Reply:
I think it's standard for a second unused param